appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/Keyframes.kt

```diff
@@ -84,8 +84,8 @@ data class Keyframes<ModelState>(
         val currentProgress = this.progress.toInt()
         val progress = progress.coerceIn(0f, maxProgress)
         Logger.log("Keyframes", "$progress")
-        progressFlow.value = progress
         segmentProgress.value = progress - currentIndex
+        progressFlow.value = progress
```

Comment:
let's leave a comment to prevent this from happening in the future?

Reply:
Not just comment, we should back this up by a test